test/unit/url_view_base.cpp

```diff
@@ -294,6 +294,32 @@ struct url_view_base_test
             assert( u.path() == "/Program Files/Games/config.ini" );
         }
 
+        //----------------------------------------
+        //
+        // Target
+        //
+        //----------------------------------------
+
+        // encoded_target()
+        {
+            url_view u( "http://www.example.com/index.html?query#frag" );
+
+            assert( u.encoded_target() == "/index.html?query" );
+        }
+
+        //----------------------------------------
+        //
+        // Resource
```

Comment:
uhhh don't add top level sections like this just put it with the other compound fields..

Reply:
Which one? Path? 

I couldn't find a good compound field to put that because this would be the compound field with path+query, but I don't think we want to remove path and query to have a single resource section.

Reply:
put it under path

Reply:
Make a new section below everything, without a name just give it a line to divide it